src/main/java/com/stripe/model/TransferReversalCollection.java

```diff
@@ -1,71 +1,92 @@
+   * </code> parameters to page through additional reversals.
    */
   public TransferReversalCollection list(Map<String, Object> params) throws StripeException {
-    return list(params, null);
+    return list(params, (RequestOptions) null);
   }
 
   /**
-   * List all reversals.
+   * You can see a list of the reversals belonging to a specific transfer. Note that the 10 most
+   * recent reversals are always available by default on the transfer object. If you need more than
+   * those 10, you can use this API method and the <code>limit</code> and <code>starting_after
+   * </code> parameters to page through additional reversals.
    */
   public TransferReversalCollection list(Map<String, Object> params, RequestOptions options)
       throws StripeException {
     String url = String.format("%s%s", Stripe.getApiBase(), this.getUrl());
     return ApiResource.requestCollection(url, params, TransferReversalCollection.class, options);
   }
-  // </editor-fold>
 
-  // <editor-fold desc="retrieve">
   /**
-   * Retrieve a reversal.
+   * By default, you can see the 10 most recent reversals stored directly on the transfer object,
+   * but you can also retrieve details about a specific reversal stored on the transfer.
    */
-  public Reversal retrieve(String id) throws StripeException {
-    return retrieve(id, (RequestOptions) null);
+  public TransferReversal retrieve(String id) throws StripeException {
+    return retrieve(id, (Map<String, Object>) null, (RequestOptions) null);
   }
 
   /**
-   * Retrieve a reversal.
+   * By default, you can see the 10 most recent reversals stored directly on the transfer object,
+   * but you can also retrieve details about a specific reversal stored on the transfer.
    */
-  public Reversal retrieve(String id, RequestOptions options) throws StripeException {
-    return retrieve(id, null, options);
+  public TransferReversal retrieve(String id, RequestOptions options) throws StripeException {
+    return retrieve(id, (Map<String, Object>) null, options);
   }
 
   /**
-   * Retrieve a reversal.
+   * By default, you can see the 10 most recent reversals stored directly on the transfer object,
+   * but you can also retrieve details about a specific reversal stored on the transfer.
    */
-  public Reversal retrieve(String id, Map<String, Object> params, RequestOptions options)
+  public TransferReversal retrieve(String id, Map<String, Object> params, RequestOptions options)
       throws StripeException {
-    String url = String.format("%s%s/%s", Stripe.getApiBase(), this.getUrl(), id);
-    return ApiResource.request(ApiResource.RequestMethod.GET, url, params, Reversal.class, options);
+    String url =
+        String.format("%s%s", Stripe.getApiBase(), String.format("%s/%s", this.getUrl(), id));
+    return ApiResource.request(
+        ApiResource.RequestMethod.GET, url, params, TransferReversal.class, options);
```

Comment:
6) Retrieve (standard): Instance method to retrieve sub-resource on collection, 
relying on collection `url` to identify the parent resource, and taking in sub-resource ID method argument.

Reply:
https://github.com/stripe/stripe-java/blob/2aa5db5c39dc6e91228507798083f373904cc8c0/src/main/java/com/stripe/model/TransferReversalCollection.java#L81-L91